packages/financial-templates-lib/src/price-feed/URTHPriceFeed.ts

```diff
@@ -0,0 +1,192 @@
+import { BN } from "../types";
```

Comment:
I would suggest changing the name from `URTHPriceFeed.ts` to `PolygonApiPriceFeed.ts`. Also, changing URTH to PolygonApi elsewhere as well e.g. in config.type. That is more apt, and shows our intent of using this price feed for other synths in future.